src/components/Filters/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './styles.css';
+import { Switch, Route, Link } from 'react-router-dom';
+
+const Listar = () => <h1>Listar</h1>;
+const Adicionar = () => <h1>Adicionar</h1>;
+
+export function Filters() {
+  return (
+    <Switch>
+      <Route exact path="/listar">
+        <Listar />
+      </Route>
+      <Route path="/adicionar">
+        <Adicionar />
+      </Route>
+      <div className="div-button">
+        <button className="button">
```

Comment:
I believe that for this issue, you don't need a `button` tag because they are **navigation links**. It's enough a `Link`. I think this is the reason: page titles can't be rendered (or you can't see them). I couldn't see them. 😭

If you need some styles in your links, you can append your classnames: 

`<Link className="button link-button" to="/list">`

Reply:
Yes, please! Link as a button. (Links with button styles if you want to do it, of course, 😄 but it's **not required** later you will work more CSS, colors, typography, and so on- In the future 🚀 )